scripts/exglobal_atmos_sfcanl.sh

```diff
@@ -134,87 +134,87 @@ fi
 
 ##############################################################
 # Output files
-if [ $DONST = "YES" ]; then
-   $NLN $DTFANL dtfanl
+if [[ ${DONST} = "YES" ]]; then
+   ${NLN} ${DTFANL} dtfanl
 fi
 
 
 ##############################################################
 # Update surface fields in the FV3 restart's using global_cycle
-mkdir -p $COMOUT/RESTART
+mkdir -p ${COMOUT}/RESTART
 
 # Global cycle requires these files
-export FNTSFA=${FNTSFA:-$COMIN_OBS/${OPREFIX}rtgssthr.grb}
-export FNACNA=${FNACNA:-$COMIN_OBS/${OPREFIX}seaice.5min.blend.grb}
-export FNSNOA=${FNSNOA:-$COMIN_OBS/${OPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
-[[ ! -f $FNSNOA ]] && export FNSNOA="$COMIN_OBS/${OPREFIX}snogrb_t1534.3072.1536"
-FNSNOG=${FNSNOG:-$COMIN_GES_OBS/${GPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
-[[ ! -f $FNSNOG ]] && FNSNOG="$COMIN_GES_OBS/${GPREFIX}snogrb_t1534.3072.1536"
+export FNTSFA=${FNTSFA:-${COMIN_OBS}/${OPREFIX}rtgssthr.grb}
+export FNACNA=${FNACNA:-${COMIN}/${OPREFIX}seaice.5min.blend.grb}
+export FNSNOA=${FNSNOA:-${COMIN}/${OPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
+[[ ! -f ${FNSNOA} ]] && export FNSNOA="${COMIN_OBS}/${OPREFIX}snogrb_t1534.3072.1536"
+FNSNOG=${FNSNOG:-${COMIN_GES_OBS}/${GPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
```

Comment:
Shouldn't this be looking in `COMIN_GES` since we are retaining this file in the `atmos` component directory?

Reply:
It looks like the logic is to access these files in COMIN. Maybe there is no need to config the COMIN_GES.